.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous Integration
+
+on:
+  schedule:
+    - cron: '0 0 1,15 * *' # At 00:00 on day-of-month 1 and 15
+
+env:
+  NODE_OPTIONS: --max-old-space-size=4096
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [test, lint]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
Ditto on explicitly specifying the Node.js version.

Reply:
Same case as above